### PR TITLE
Moved ColorPicker to FC.

### PR DESCRIPTION
**Pull request by NitzanWix**
## Description
Created FC of the ColorPicker.

## Changelog
ColorPicker - Changed to FC.

### CODE REVIEW

File: src/components/colorPicker/index.tsx [hunk 1]
```diff
@@ -58,78 +58,65 @@ const ACCESSIBILITY_LABELS = {
...
 
-  static defaultProps = {
-    accessibilityLabels: ACCESSIBILITY_LABELS,
-    backgroundColor: Colors.$backgroundDefault
-  };
-
-  state = {
-    show: false
-  };
+  const showDialog = () => setShow(true);
```
Comment (ethanshar) [inline src/components/colorPicker/index.tsx:74]: wrap with useCallback

File: src/components/colorPicker/index.tsx [hunk 2]
```diff
@@ -58,78 +58,65 @@ const ACCESSIBILITY_LABELS = {
...
 
-  get animatedIndex() {
-    const {animatedIndex, colors} = this.props;
-    if (animatedIndex === undefined) {
-      return colors.length - 1;
-    }
-    return animatedIndex;
-  }
+  const hideDialog = () => setShow(false);
```
Comment (ethanshar) [inline src/components/colorPicker/index.tsx:76]: wrap with useCallback

**Review by ethanshar — COMMENTED** [review]
Looks good, wrote small comments. Also you need to resolve conflicts

File: src/components/colorPicker/index.tsx [hunk 3]
```diff
@@ -58,77 +58,65 @@ const ACCESSIBILITY_LABELS = {
...
-    backgroundColor: Colors.$backgroundDefault
-  };
-
-  state = {
-    show: false
-  };
+  const showDialog = useCallback(() => setShow(true), []);
 
-  get animatedIndex() {
-    const {animatedIndex, colors} = this.props;
```
Comment (Inbal-Tish) [inline src/components/colorPicker/index.tsx:74]: Why this logic was deleted?

Comment (NitzanWix) [reply src/components/colorPicker/index.tsx:74]:
```
get animatedIndex() {
    const {animatedIndex, colors} = this.props;
    if (animatedIndex === undefined) {
      return colors.length - 1;
    }
    return animatedIndex;
  }
```

File: src/components/colorPicker/index.tsx [hunk 4]
```diff
@@ -58,77 +58,65 @@ const ACCESSIBILITY_LABELS = {
...
-            input: accessibilityLabels?.input
-          }}
+  return (
+    <View row testID={testID} style={{backgroundColor}}>
+      <ColorPalette
+        value={value}
+        colors={colors}
+        style={styles.palette}
+        usePagination={false}
+        animatedIndex={animatedIndex === undefined ? colors.length - 1 : animatedIndex}
```
Comment (NitzanWix) [inline src/components/colorPicker/index.tsx:85]: Changed this to a ternary to have the same logic.

File: src/components/colorPicker/index.tsx [hunk 5]
```diff
@@ -58,77 +58,65 @@ const ACCESSIBILITY_LABELS = {
...
-            input: accessibilityLabels?.input
-          }}
+  return (
+    <View row testID={testID} style={{backgroundColor}}>
+      <ColorPalette
+        value={value}
+        colors={colors}
+        style={styles.palette}
+        usePagination={false}
+        animatedIndex={animatedIndex ?? colors.length - 1}
```
Comment (NitzanWix) [inline src/components/colorPicker/index.tsx:85]: I changed the fallback to this. (Using || will be wrong when the index is 0)

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**